libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -1210,6 +1244,65 @@ export const buildTigerSpecificFunctions = (
             });
         });
     },
+
+    getWorkspaceDataFilter: async (
+        workspaceId: string,
+        objectId: string,
+    ): Promise<WorkspaceDataFilterResult> => {
+        try {
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.getEntityWorkspaceDataFilters({
+                    workspaceId,
+                    objectId,
+                });
+                return result.data;
+            });
+        } catch (error: any) {
+            throw convertApiError(error);
+        }
+    },
+
+    setWorkspaceDataFilter: async (
+        workspaceId: string,
+        workspaceDataFilter: WorkspaceDataFilter,
+    ): Promise<void> => {
+        return authApiCall(async (sdk) => {
+            await sdk.entities.createEntityWorkspaceDataFilters({
+                workspaceId,
+                jsonApiWorkspaceDataFilterInDocument: workspaceDataFilter,
+            });
+        });
+    },
+
+    getWorkspaceDataFilterSetting: async (
+        workspaceId: string,
+        objectId: string,
+    ): Promise<WorkspaceDataFilterSettingResult> => {
+        try {
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.getEntityWorkspaceDataFilterSettings({
+                    workspaceId,
+                    objectId,
+                });
+                return result.data;
+            });
+        } catch (error: any) {
+            throw convertApiError(error);
+        }
+    },
+
+    setWorkspaceDataFilterSetting: async (
+        workspaceId: string,
+        workspaceDataFilterSettings: WorkspaceDataFilterSetting,
```

Comment:
Please rename `workspaceDataFilterSettings` → `workspaceDataFilterSetting` (in singular).

A name in plural might make people think that it is an array.